register-autonomoustuff-repository/action.yaml

```
@@ -0,0 +1,26 @@
+name: register-autonomoustuff-repository
+description: Register AutonomouStuff repositry into rosdep
+
+inputs:
+  rosdistro:
+    description: ROS distro
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # Fetch with depth=0 to push to gh-pages
+        fetch-depth: 0
+
```

I think this part isn't necessary.

```suggestion
    - name: Checkout repository
      uses: actions/checkout@v2
      with:
        # Fetch with depth=0 to push to gh-pages
        fetch-depth: 0
```

---

Ah, no. The entier `Checkout repository` isn't necessary.

---

My suggestion format was wrong? I'm sorry. :smiling_face_with_tear:

---

Fixed at https://github.com/autowarefoundation/autoware-github-actions/pull/6/commits/3fd62e66b8a9ad9395fc9b3a1f994b9404c732a3